cms/signals/__init__.py

```diff
@@ -26,10 +27,21 @@
 post_unpublish = Signal(providing_args=["instance", "language"])
 
 # fired if a public page with an apphook is added or changed
-urls_need_reloading = Signal(providing_args=[])
+cms_urls_need_reloading = Signal(providing_args=[])
```

Comment:
I think we should not change the signal name without a deprecation period, and in a patch release